libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.ts

```diff
@@ -5,4 +5,4 @@ import { Component } from '@angular/core';
   templateUrl: './home-page.component.html',
   styleUrls: ['./home-page.component.scss'],
 })
-export class HomePageComponent {}
+export class HomePageComponent {}
```

Comment:
This entire component needs to be inside a components folder on this lib..

Sync your fork, the components folder was already created.